client/lib/signup/step-actions.js

```diff
@@ -132,7 +133,7 @@ export function createSiteWithCart(
 				siteGoals: siteGoals || undefined,
 			},
 			validate: false,
-			find_available_url: isPurchasingItem,
+			find_available_url: isPurchasingItem || 'from-site' === flowName,
```

Comment:
I'm sure you've already thought of this, but do we need to check that `importUrl` was explicitly given as we do [here](https://github.com/Automattic/wp-calypso/pull/27401/files#diff-df013e2a4c6a6c2b5973d50e326984f8R71)?

As in, would there be any issue with setting this to `true` in cases where the `flowName` is `from-site`, but no `importUrl` is given?

Reply:
Good point -- let's add a check here too.